pants-plugins/tests/python/internal_backend_test/sitegen/test_sitegen.py

```diff
@@ -204,7 +205,13 @@ def test_site_toc(self):
                                    {{/site_toc}}
                                    """)
     self.assertIn("DEPTH=1 LINK=None HEADING=non_collapse", rendered)
-    self.assertIn(u"DEPTH=1 LINK=[{'text': u'Page 2: Electric Boogaloo', 'link': u'subdir/page2.html', 'here': False}, {'text': u'Pants Build System', 'link': u'index.html', 'here': True}] HEADING=collapse", rendered)
+    # Py2 and Py3 order the elements differently and Py3 doesn't render 'u' in unicode literals. Both are valid.
```

Comment:
I'm pretty sure that neither of these orderings _should_ be deterministic :S

But, if the tests are solid...